src/api/goods.js

```diff
@@ -3,9 +3,21 @@ const API_URL = `https://mate-academy.github.io/react_dynamic-list-of-goods/good
 
 export function getAll() {
   return fetch(API_URL)
-    .then(response => response.json());
+    .then(response => response.json())
+    .then(data => data);
 }
 
-export const get5First = () => {};
+export const get5First = async() => {
+  const goods = await getAll();
+  const sortedGoods = goods.sort((a, b) => (
+    a.name.localeCompare(b.name)
+  ));
```

Comment:
тебе достаточно использовать async и await в одном месте, как ты это делаешь в App. Здесь же будет хватать вот такого кода
```suggestion
export const get5First = () => {
  return getAll()
    .then(goods => goods
      .sort((a, b) => (
        a.name.localeCompare(b.name)
    )));
```

Reply:
не совсем понимаю, чем .then(goods => goods лучше, чем async и await, поэтому пока оставлю, как было)

Reply:
Ты же используешь их уже в App. Они не лучше, но у тебя дублирование кода получается

Reply:
they are both the same, just different syntax